.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "stage": 1,
+  "loose": true,
+  "optional"  : [
+    "utility.inlineEnvironmentVariables"
+  ]
+}
```

Comment:
You can pass them into babel-loader using "query-string" style in the webpack config but I like keeping the configuration in .babelrc. It seems to be the standard way.